arches/app/media/js/utils/set-csrf-token.js

```diff
@@ -1,4 +1,4 @@
-define(['jquery', 'js-cookie'], function ($, Cookies) {
```

Comment:
This is required in `javascript.htm` which is loaded in `base.html` and other templates. When webpacking jquery is already loaded and this component will therefore throw an error if it's loaded again